Integration/Sources/ViewController.swift

```diff
@@ -29,8 +29,8 @@ class ViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         let formatter = Formatter()
-        DDTTYLogger.sharedInstance.logFormatter = formatter
-        DDLog.add(DDTTYLogger.sharedInstance)
+        DDOSLogger.sharedInstance.logFormatter = formatter
+        DDLog.add(DDOSLogger.sharedInstance)
```

Comment:
I might have accidentally fixed that already in #968...

Reply:
No problem :)